playground/vanilla/src/main.ts

```diff
@@ -12,14 +12,24 @@ console.log(headers2.constructor.name)
 const Storyblok = new StoryblokClient({
   accessToken: import.meta.env.VITE_ACCESS_TOKEN as string,
   headers,
+  version: 'draft',
 })
 
 try {
   const result = await Storyblok.get('cdn/stories/', {
-    version: 'draft',
     resolve_relations: 'root.author',
   })
 
+  const links = await Storyblok.getAll('cdn/links', {
+    version: 'published',
+  })
+
+  console.log({
+    links,
+    result,
+  })
```

Comment:
probably a leftover?

Reply:
I removed it but consider that the playground is for us to test, not an actual maintained example